_blog/deploying-laravel-to-kubernetes/index.md

```diff
@@ -0,0 +1,354 @@
+---
+layout: post
+title: Deploying Laravel to Kubernetes
+date: 2018-02-18 00:00:00
+categories: kubernetes laravel php
+author: "Keith Mifsud"
+author_avatar: "keith_mifsud.jpg"
+image: /blog/deploying-laravel-to-kubernetes/laravel-and-kubernetes.png
+
+---
+
+As most of us know, Laravel is a great framework for developing any PHP application. Whether we need to   prototype a new idea, develop an MVP (Minimum Viable Product) or release a 
+full-fledged enterprise system, Laravel facilitates all of the development 
+tasks and work-flows.
+
+How we deal with deploying the application is another story. Vagrant is 
+very good with setting up a local environment similar to a remote server, 
+however, in production, we will most likely require more than just one 
+web host and one database. We probably will have separate services for several 
+requirements. We also need to have mechanisms in place to ensure that our 
+application is always online and that the servers can effectively balance the
+ load.
+
+In this article, I'll explain how to deal with the simple requirement of
+ running a Laravel application across three Nodes on a Kubernetes architecture. 
```

Comment:
Minikube runs on a single node. You can think about a node as a VM. Inside a node (VM), you have the Docker daemon where you can run Docker containers. You can launch as many containers as you wish, as long as you have enough memory/CPU. In this particular tutorial, you have one single node, but you're starting 3 replicas.
Which translated from Kubernetes jargon becomes: You have 1 VM with 3 Docker containers running on it.

Reply:
I like the way you simplified the explanation :)